editor/src/components/canvas/controls/formula-bar.tsx

```diff
@@ -106,6 +111,26 @@ export const FormulaBar = betterReactMemo('FormulaBar', () => {
     isFeatureEnabled('TopMenu ClassNames') && selectedElement != null && selectedMode === 'css'
   const inputFieldVisible = !classNameFieldVisible
 
+  const editorStoreRef = useRefEditorState((store) => store)
+  const onKeyDown = React.useCallback(
+    (event: React.KeyboardEvent<HTMLElement>) => {
+      const namesByKey = applyShortcutConfigurationToDefaults(
+        editorStoreRef.current.userState.shortcutConfig,
+      )
+      handleShortcuts(namesByKey, event.nativeEvent, null, {
+        [TOGGLE_FOCUSED_OMNIBOX_TAB]: () => {
+          event.stopPropagation()
+          event.preventDefault()
```

Comment:
I don't _think_ this is necessary anymore, because it's in `handleShortcuts`.

Reply:
Ah good catch. It is indeed unnecessary, thanks!